crates/server/src/api/proof.rs

```diff
@@ -74,6 +76,7 @@ async fn prove_consistency(
     State(config): State<Config>,
     Json(body): Json<ConsistencyRequest>,
 ) -> Result<Json<ConsistencyResponse>, ProofApiError> {
+    println!("MADE IT TO HANDLER");
```

Comment:
Remove

Reply:
Nice, I've used it before, but haven't formed a habit of it yet.  Will definitely play with it some more though.  Deleted the `println!`.